src/backend/gl/src/window/egl.rs

```diff
@@ -596,9 +613,6 @@ impl Surface {
         gl.bind_framebuffer(glow::READ_FRAMEBUFFER, None);
 
         self.egl.swap_buffers(self.display, self.raw).unwrap();
-        self.egl
```

Comment:
I'm not sure about these changes. I don't know why we should make the current context without surface every frame?

Reply:
It's just returning everything back to what it is before the `present()` is called. We could technically leave the surface bound, but it could introduce accidental code errors when something is rendering to the default framebuffer, and we aren't catching a GL error on that (instead corrupting one of the swapchains). Note that the `self.context` can work with different surfaces, not just one.

Reply:
Any ideas how we can get around this limitation?

Reply:
Yep, we can keep the `PBuffer` around and use it here when calling `make_current`